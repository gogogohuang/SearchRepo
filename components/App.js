import React, { Component } from 'react';
import styled from 'styled-components';

import { createGlobalStyle } from "styled-components";
import SearchContainer from './search/searchContainer';

const GlobalStyle = createGlobalStyle`
  @import url('https://fonts.googleapis.com/css?family=Raleway');
  body {
    font-family: 'Raleway', sans-serif;
    margin: 0;
    overflow-x: hidden;
  }
`;

const Container = styled.div`
  width: 100vw;
  position: relative;
`;

class App extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
        <Container>
          <GlobalStyle />
          <SearchContainer />
        </Container>
    );
  }
}

export default App;
