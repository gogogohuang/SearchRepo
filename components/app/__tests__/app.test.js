import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';
import 'jest-enzyme';
import 'jest-styled-components';
import 'babel-polyfill';

describe('App', () => {
  let makeSubject;

  beforeEach(() => {
    jest.resetModules();
    makeSubject = () => shallow(<App />)
  })
  it('should render as snapshot', () => {
    const subject = makeSubject();
    expect(subject.debug()).toMatchSnapshot();
  });
})
