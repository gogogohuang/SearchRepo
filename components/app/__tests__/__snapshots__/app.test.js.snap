// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render as snapshot 1`] = `
"<styled.div>
  <GlobalStyleComponent />
  <SearchContainer />
</styled.div>"
`;
